src/LearningObjects/LearningObjectRouteHandler.ts

```diff
@@ -168,10 +168,25 @@ export function initializePrivate({
     }
   };
 
+  const getLearningObjectChildren = async (req: Request, res: Response) => {
+    try{
+      const id = req.params.id;
+      const children = await LearningObjectInteractor.getLearningObjectChildrenById(
+        dataStore, 
+        id
+      );
+      res.status(200).send(children); 
+    } catch (e) {
+      console.error(e);
+      res.status(500).send(e); 
```

Comment:
This route should not always throw a 500 when an error occurs. Status code 500 indicates that the server itself is at fault. This may not always be the case. We want to map the error object to an appropriate status code in order to provide the client with more context. Luckily, this function already exists.

```suggestion
      if (e instanceof Error) {
        const status = mapErrorToStatusCode(e);
        res.status(status.code).json({message: status.message});
      } else {
        res.sendStatus(500);
      }
```